embedded/src/main/java/org/wildfly/core/embedded/logging/EmbeddedLogger.java

```diff
@@ -51,86 +47,6 @@
-     * closed.
-     *
-     * @param cause the cause of the error.
-     * @param file  the file.
-     */
-    @LogMessage(level = WARN)
-    @Message(id = 1, value = "Could not close handle to mounted %s")
-    void cannotCloseFile(@Cause Throwable cause, VirtualFile file);
-
-    /**
-     * Logs a warning message indicating the class file, represented by the {@code file} parameter, could not be loaded.
-     *
-     * @param cause the cause of the error.
-     * @param file  the file that could not be loaded.
-     */
-    @LogMessage(level = WARN)
-    @Message(id = 2, value = "Could not load class file %s")
-    void cannotLoadClassFile(@Cause Throwable cause, VirtualFile file);
-
-    /**
-     * Logs a warning message indicating there was an exception closing the file.
-     *
-     * @param cause the cause of the error.
-     * @param file  the file that failed to close.
-     */
-    @LogMessage(level = WARN)
-    @Message(id = 3, value = "Exception closing file %s")
-    void errorClosingFile(@Cause Throwable cause, VirtualFile file);
-
-    /**
-     * Logs a warning message indicating there was a failure to undeploy the file.
-     *
-     * @param cause the cause of the error.
-     * @param file  the file that failed to undeploy.
-     */
-    @LogMessage(level = WARN)
-    @Message(id = 4, value = "Failed to undeploy %s")
-    void failedToUndeploy(@Cause Throwable cause, File file);
-
-    /**
-     * Logs a warning message indicating the file on the ClassPath could not be found.
-     *
-     * @param file the file that could not be found.
-     */
-    @LogMessage(level = WARN)
-    @Message(id = 5, value = "File on ClassPath could not be found: %s")
-    void fileNotFound(VirtualFile file);
-
-    /**
-     * Logs a warning message indicating an unknown file type was encountered and is being skipped.
-     *
-     * @param file the file.
-     */
-    @LogMessage(level = WARN)
-    @Message(id = 6, value = "Encountered unknown file type, skipping: %s")
-    void skippingUnknownFileType(VirtualFile file);
-
-
-    /**
-     * Creates an exception indicating the file, represented by the {@code fileName} parameter, could not be mounted.
-     */
-    @Message(id = 7, value = "Could not mount file '%s'")
-    RuntimeException cannotMountFile(@Cause Throwable cause, String fileName);
-
-    /**
-     * Creates an exception indicating the contents of the file could not be read.
-     */
-    @Message(id = 8, value = "Could not read contents of %s")
-    RuntimeException cannotReadContent(@Cause Throwable cause, VirtualFile file);
-
-    /**
-     */
-    @Message(id = 9, value = "One or more exclusion values must be specified")
-    IllegalArgumentException exclusionValuesRequired();
-
-    //@Message(id = 10, value = "WARNING: Failed to load the specified logmodule %s")
-    //String failedToLoadLogModule(ModuleIdentifier moduleId);
-
-    /**
```

Comment:
Please don't delete log methods. Just comment them out so the reason for gaps in the list of ids is clearly visible.